### DISCUSSION TITLE
How to test the accuracy of a DP model

### DISCUSSION THREAD

**Discussion by murpphy** [deepmodeling/dpgen#725] [Q&A]
Hello,developers
After watching the instructional videos and reading many literatures, I don't know how to test the model trained by DPGEN. It seems that autotest does not include all the test items in the literature. Could you please tell me how you do the test? I will be overwhelmed gratitude.
I saw the "DP Little Story" posted on the WeChat public account. Linfeng Zhang and Jinzhe Zeng mentioned this issue in their emails, in which DFT and DP energy/force analysis were performed.

**Comment by AnguseZhang** [comment]
Which test do you want to perform? If you want to compare the difference between DFT and DP, you can use `dp test`.

**Reply by murpphy** [reply]
Thanks for your answer, I will try to do it.

**Reply by murpphy** [reply]
Where can I study the dp test, such as instructional videos or literature.

**Reply by murpphy** [reply]
Hello, the results after I use dp test are shown in the figure, the prediction effect of energy is very poor, does this mean that there is a problem with the training process?

![1](https://user-images.githubusercontent.com/102002524/167747920-d61769a9-4a16-4348-ad23-dfd6b0b412ef.png)

**Reply by AnguseZhang** [reply]
Probably there are problems with your DFT calculations or training settings.

**Reply by murpphy** [reply]
Thank you, I'm trying to adjust my training settings, but I still have a problem that I don't understand, I didn't find the test dataset for DPGEN, the init_bulk only has the initial training dataset. If I want to do testing, how should this test dataset be prepared?
